p11mod/p11mod.go

```diff
@@ -391,9 +395,16 @@ func (ll *llBackend) Login(sh pkcs11.SessionHandle, userType uint, pin string) e
 }
 
 func (ll *llBackend) Logout(sh pkcs11.SessionHandle) error {
-	// TODO
-	log.Println("p11mod Logout: not implemented")
-	return nil
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return err
+	}
+
+	if trace {
+		log.Println("p11mod Logout user ")
```

Comment:
Please remove the ` user ` suffix; it's unnecessary.

Reply:
Sorry, I did not notice I added the user suffix 
I would do that asap :)